Detectors/TRD/calibration/workflow/TRDDCSDataProcessorSpec.h

```diff
@@ -116,16 +128,40 @@ class TRDDCSDataProcessor : public o2::framework::Task
     }
 
     mProcessor = std::make_unique<o2::trd::DCSProcessor>();
-    int verbosity = ic.options().get<bool>("processor-verbosity");
+    int verbosity = ic.options().get<int>("processor-verbosity");
     if (verbosity > 0) {
       LOG(info) << "Using verbose mode for TRD DCS processor";
       mProcessor->setVerbosity(verbosity);
     }
+
+    // LB: set maximum number of alarms in change in FedChamberStatus and FedCFGtag
+    int alarmfed = ic.options().get<int>("DPs-max-counter-alarm-fed");
+    if (alarmfed > 0) {
```

Comment:
I would also allow zero so that no alarm is printed

Reply:
Done! See commit https://github.com/AliceO2Group/AliceO2/pull/12079/commits/d5011ff8800ba4fa5d17263e1931ef2d57f3b0bc.